moped-editor/src/views/projects/projectsListView/ProjectsListViewFiltersConf.js

```diff
@@ -36,6 +36,20 @@ export const OPERATORS_WITHOUT_SEARCH_VALUES = [
  */
 export const PROJECT_LIST_VIEW_FILTERS_CONFIG = {
   fields: [
+    {
+      name: "project_name_full",
+      label: "Full name",
+      placeholder: "Enter full project name",
```

Comment:
Unrelated but i don't think we use this placeholder value anywhere. Might be nice to create a tech debt snackoo to look into this.

Reply:
I'll make a snackoo! this feels like a holdover from old gql abstract configs

Reply:
https://github.com/cityofaustin/atd-data-tech/issues/16783